src/serve_rendered.js

```diff
@@ -1374,6 +1374,9 @@ export const serve_rendered = {
       type: 'baselayer',
     };
     const attributionOverride = params.tilejson && params.tilejson.attribution;
+    if (styleJSON.center && styleJSON.zoom) {
+      tileJSON.center = styleJSON.center.concat(styleJSON.zoom);
```

Comment:
I was going to merge this, but I had issues when the zoom level was a decimal value.
![image](https://user-images.githubusercontent.com/3792408/228118875-ba481208-f93f-4226-83c1-51544f19ddbc.png)

I figured out if I added a round Math.round aound styleJSON.zoom it fixes the issue
![image](https://user-images.githubusercontent.com/3792408/228119202-6a351502-2611-4d79-82ab-d68308589382.png)

Reply:
`tileJSON.center = styleJSON.center.concat(Math.round(styleJSON.zoom));` worked